src/server/services/transformers/service-to-entity-data-list.test.js

```diff
@@ -17,7 +17,7 @@ describe('#serviceToEntityDataList', () => {
             kind: 'link',
             newWindow: true,
             url: `https://github.com/${githubOrg}/cdp-deployables`,
-            value: `${githubOrg}/cdp-deployables`
+            value: 'DEFRA/cdp-deployables'
```

Comment:
should we remove the `DEFRA` here too?

Reply:
No, well, ideally, but it was the one case were the repo does not match the service name.

Reply:
This whole test is an edge case that should not be possible to reach any more